docs/src/userguide/extension_types.rst

```diff
@@ -782,6 +1016,12 @@ declaration makes an extension type defined in external C code available to a
 Cython module. A public extension type declaration makes an extension type
 defined in a Cython module available to external C code.
 
+.. note::
+
+    Cython currently does not support Extension types declared as extern or public
+    in Pure Python mode.
```

Comment:
```suggestion
    Cython currently does not support Extension types declared as extern or public
    in Pure Python mode.  This is not considered an issue since public/extern extension
    types are most commonly declared in `.pxd` files and not in `.py` files.
```